x-pack/plugin/ent-search/src/test/java/org/elasticsearch/xpack/application/analytics/action/DeleteAnalyticsCollectionRequestSerializingTests.java

```diff
@@ -7,12 +7,14 @@
 
 package org.elasticsearch.xpack.application.analytics.action;
 
+import org.elasticsearch.TransportVersion;
 import org.elasticsearch.common.io.stream.Writeable;
-import org.elasticsearch.test.AbstractWireSerializingTestCase;
+import org.elasticsearch.xcontent.XContentParser;
+import org.elasticsearch.xpack.core.ml.AbstractBWCSerializationTestCase;
 
 import java.io.IOException;
 
-public class DeleteAnalyticsCollectionRequestSerializingTests extends AbstractWireSerializingTestCase<
+public class DeleteAnalyticsCollectionRequestSerializingTests extends AbstractBWCSerializationTestCase<
```

Comment:
Nitpick: Should we refactor `*SerializingTests` to `*BWCSerializingTests` to be clear?

Reply:
Yes. I think it make sense. Will do 👍